ddp-workspace/projects/ddp-singular/src/app/components/activities-list/activities-list.component.ts

```diff
@@ -11,6 +11,7 @@ import { ActivityIcons } from '../../constants/activity-icons';
 })
 export class ActivitiesListComponent {
   @Input() activities: ActivityInstance[];
+  @Input() disableButtons = false;
```

Comment:
Since the "action" (e.g. getting an activity) happens in the parent component, we use @Input so parent can tell this child "disable buttons"